<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connection</class>
 <widget class="QDialog" name="connection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conexão</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="name">
     <property name="placeholderText">
      <string>Nome da Conexão</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="host">
     <property name="placeholderText">
      <string>Host</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="database">
     <property name="placeholderText">
      <string>Base de Dados</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="username">
     <property name="placeholderText">
      <string>Usuário</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="password">
     <property name="inputMask">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Senha</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="port">
     <property name="toolTip">
      <string>Porta TCP</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>65000</number>
     </property>
     <property name="value">
      <number>3306</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>connection</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
